runtime/tests/checker/typeargument_test.go

```diff
@@ -178,6 +155,681 @@ func TestCheckTypeArguments(t *testing.T) {
 	})
 }
 
+func TestCheckParameterizedTypeIsInstantiated(t *testing.T) {
+
+	t.Parallel()
+
+	baseValueActivation := sema.NewVariableActivation(sema.BaseValueActivation)
+	baseValueActivation.DeclareValue(stdlib.InclusiveRangeConstructorFunction)
+	options := ParseAndCheckOptions{
+		Config: &sema.Config{
+			BaseValueActivation: baseValueActivation,
+		},
+	}
```

Comment:
Try making this a function: 

```go
test := func(t *testing.T, code) error {
	baseValueActivation := sema.NewVariableActivation(sema.BaseValueActivation)
	baseValueActivation.DeclareValue(stdlib.InclusiveRangeConstructorFunction)
	options := ParseAndCheckOptions{
		Config: &sema.Config{
			BaseValueActivation: baseValueActivation,
		},
	}

	_ , err := ParseAndCheckWithOptions(t, code, options)
	return err
}
```

That will likely fix the reported races

Reply:
Thanks for the suggestion! Done.